arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/stream/concurrent/PubSub.kt

```diff
@@ -32,7 +32,7 @@ internal interface Publish<A> {
    * Evaluates to `false` if element was not published.
    * Evaluates to `true` if element was published successfully.
    */
-  suspend fun tryPublish(a: A): Boolean
+  fun tryPublish(a: A): Boolean
```

Comment:
Why was it suspend?

Reply:
Because the underlying implementation was still relying on `suspension`, which has now been rewritten to work without suspension.